src/amo/components/Header/index.js

```diff
@@ -68,132 +69,146 @@ export class HeaderBase extends React.Component {
     );
 
     return (
-      <header className="Header">
-        <div className="Header-content">
-          {isHomePage ? (
-            <h1 className="Header-title-wrapper">{headerLink}</h1>
-          ) : (
-            headerLink
-          )}
-        </div>
-
-        <SectionLinks className="Header-SectionLinks" location={location} />
-
-        <div className="Header-user-and-external-links">
-          <Link
-            className="Header-extension-workshop-link Header-button"
-            href={`${_config.get(
-              'extensionWorkshopUrl',
-            )}/${makeQueryStringWithUTM({
-              utm_content: 'header-link',
-              utm_campaign: null,
-            })}`}
-            external
-            prependClientApp={false}
-            prependLang={false}
-            target="_blank"
-            title={i18n.gettext('Learn how to create extensions and themes')}
-          >
-            {i18n.gettext('Extension Workshop')}
-          </Link>
-          <Link
-            className="Header-developer-hub-link Header-button"
-            href="/developers/"
-            external
-            prependClientApp={false}
-            target="_blank"
-            title={i18n.gettext('Submit and manage extensions and themes')}
-          >
-            {i18n.gettext('Developer Hub')}
-          </Link>
-          <GetFirefoxButton
-            buttonType={GET_FIREFOX_BUTTON_TYPE_HEADER}
-            className="Header-download-button Header-button"
-          />
-
-          {siteUser ? (
-            <DropdownMenu
-              text={siteUser.name}
-              className="Header-authenticate-button Header-button"
+      <header
+        className={makeClassName(
+          'Header',
+          _config.get('enableFeatureHeroRecommendation')
+            ? 'Header-new-hero'
```

Comment:
I suggest avoiding "old" and "new" for feature flags because names have a tendency to stick around -- i.e. we will likely forget to update them. I suggest `Header-no-hero-promo` for the old one and `Header-hero-promo` for the new one.